src/Sulu/Bundle/AdminBundle/Resources/js/components/Popover/popover.scss

```diff
@@ -0,0 +1,31 @@
+@import '../../containers/Application/colors.scss';
+
+$popoverBorderColor: $frenchGray;
+$popoverBackgroundColor: $wildSand;
+$popoverBoxShadow: 2px 6px 12px 0 rgba($silver, .22);
+
+/* to disable scrolling */
+.container {
+    position: fixed;
+    top: 0;
+    left: 0;
+    width: 100%;
+    height: 100%;
+    z-index: 1;
+    pointer-events: none;
+}
+
+.popover {
```

Comment:
Shouldn't the component not add any styling at all, with positioning being the only exception?

Reply:
I think we might not need the entire element receiving this class at all.